sleepecg/io/physionet.py

```diff
@@ -21,8 +21,8 @@
 def _list_physionet(
     data_dir: Path,
     db_slug: str,
-    db_version: Optional[str] = "1.0.0",
-    pattern="*",
+    db_version: str = "1.0.0",
```

Comment:
I see that you enabled an option to not require `Optional`, but why is `Optional` still used in other places (like e.g. `_generate_gudb_md5()`)?

Reply:
According to the [Python docs](https://docs.python.org/3/library/typing.html#typing.Optional), `Optional` should be used if `None` is allowed. So it's appropriate in `_generate_gudb_md5`, but shouldn't be there in `download_physionet` and `_get_physionet_checksums`, I'll fix that!